installers/dhall/demo.dhall

```diff
@@ -0,0 +1,9 @@
+{ name         = "demo"
+, keyPrefix    = "TODO, default without the suffix"
+, relays       = "TODO, ip support missing"
+, updateServer = "https://disabled.iohkdev.io"
+, reportServer = "http://staging-report-server.awstest.iohkdev.io:8080"
+, installDirectorySuffix = " Demo"
+, macPackageSuffix       = "Demo"
+, walletPort             = 8092
```

Comment:
why 8092? demo cluster defaults to 8090, and I think 8092 is reserved for staging in daedalus itself when IPC isn't in play.